spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/controller/ServiceInstanceController.java

```diff
@@ -105,7 +106,16 @@ public ServiceInstanceController(CatalogService catalogService, ServiceInstanceS
 								LOG.debug("Creating a service instance succeeded: serviceInstanceId={}, response={}",
 										serviceInstanceId, response)))
 				.map(response -> new ResponseEntity<>(response, getCreateResponseCode(response)))
-				.switchIfEmpty(Mono.just(new ResponseEntity<>(HttpStatus.CREATED)));
+				.switchIfEmpty(Mono.just(new ResponseEntity<>(HttpStatus.CREATED)))
+				.onErrorResume(e -> {
+					if (e instanceof ServiceInstanceDoesNotExistException ||
```

Comment:
This should be `ServiceDefinitionDoesNotExistException`, and not `ServiceInstanceDoesNotExistException`, since we're creating a new service instance in this API.

Reply:
Thanks for noticing, I fix it.